core/update.php

```diff
@@ -617,8 +625,7 @@ function update_check_requirements($skip_warnings = FALSE) {
         install_goto('core/update.php?op=results');
       }
       else {
-        // Skip the backup and go to update selection if upgrading from Drupal 7.
-        $op = update_backup_enabled() ? 'backup' : 'selection';
```

Comment:
Removing this breaks hiding the backup step if `$settings['backup_directory'] = FALSE;`

Is removing this necessary to get the upgrade working?

Reply:
I just gave it a try and reverted that line. The upgrade still runs fine if that check is there, so it's safe to revert that.

Reply:
Thank you!